portfolio/src/main/webapp/script.js

```diff
@@ -58,11 +58,13 @@ function getRandomName() {
 }
 
 /**
- * Fetches comments from DataServlet and adds them to the DOM as a list.
+ * Fetches the last commentsLimit comments from DataServlet and adds them to the DOM as a list.
+ * commentsLimit is selected by the user and sent to the server as parameter in the query string.
  */
 function getComments() {
-  fetch('/data').then(response => response.json()).then((comments) => {
+  const dataURL = `data?commentsLimit=${document.getElementById('commentsLimit').value}`;
```

Comment:
The removed line is `/data` and the added line is `data?....` (no leading slash) - does this still work ok like this?

I think later on in your internship you'll be adding automated tests which this would hopefully check for you :-)

Reply:
It worked, but I'll add the leading slash